src/main/views/sso-user.njk

```diff
@@ -0,0 +1,60 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
+{% from "macros/csrf.njk" import csrfProtection %}
+{% from "macros/errorSummary.njk" import errorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% extends "template.njk" %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ breadcrumb([
+    ['Manage users', urls.MANAGER_USERS_URL],
+    ['Delete User']
+  ]) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    {{ errorSummary(error) }}
+
+    <form id="userRemoveSsoForm" method="POST" action="{{ urls.USER_DISABLE_SSO_URL }}">
+      {{ csrfProtection(csrfToken) }}
+      <input type="hidden" name="_userId" value="{{ content.user.id }}">
+      <div class="govuk-grid-column-one-half">
+        {{ govukRadios({
+          classes: "govuk-radios--inline",
+          idPrefix: "confirmRadio",
+          name: "confirmSso",
+          fieldset: {
+            legend: {
+              text: "Are you sure you want to remove single sign on for " + content.user.forename + " " + content.user.surname + "?",
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--l"
+            }
+          },
+          hint: {
+            text: "Are you sure you want to remove single sign on for this user? The user will need to reset their password."
```

Comment:
Should be 'single sign-on' with an hyphen

Reply:
fixed